src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -1275,6 +1283,27 @@ namespace
             alpha->setToolTip(m_params_metadata.get_path("sppm.alpha.help"));
             sublayout->addRow("Alpha:", alpha);
         }
+
+        void create_advanced_settings(QVBoxLayout* parent)
+        {
+            QGroupBox* groupbox = new QGroupBox("Advanced");
+            parent->addWidget(groupbox);
+
+            QVBoxLayout* layout = new QVBoxLayout();
+            groupbox->setLayout(layout);
+
+            create_advanced_max_ray_intensity_settings(layout);
+        }
+
+        void create_advanced_max_ray_intensity_settings(QVBoxLayout* parent)
+        {
+            QDoubleSpinBox* max_ray_intensity = create_double_input("advanced.max_ray_intensity", 0.0, 1.0e9, 3, 0.1);
```

Comment:
1.0e9? Is that really the max value for MRI in the unidirectional path tracer?

Also precision of 3 digits after the decimal place seems excessive, aren't we using 1 in UPT?

Reply:
I copied the line from the UPT and haven't modified anything